cfbs/commands.py

```diff
@@ -1000,3 +1001,74 @@ def input_command(args, input_from="cfbs input"):
         files_to_commit.append(input_path)
     config.save()
     return Result(0, do_commit, None, files_to_commit)
+
+
+def set_input_command(name, src_path):
+    config = CFBSConfig.get_instance()
+    module = config.get_module_from_build(name)
+    if module is None:
+        log.error("Module '%s' not found" % name)
+        return 1
+
+    spec = module.get("input")
+    if spec is None:
+        log.error("Module '%s' does not accept input" % name)
+        return 1
+    log.debug("Input spec for module '%s': %s" % (name, pretty(spec)))
+
+    if src_path is None:
+        try:
+            data = json.load(stdin)
```

Comment:
If the user runs the command `cfbs set-input some-module`, they will get a seemingly stuck/hanging tool. We should either add some prompt or change the stdin-based input to be based on an extra option or a special value of the argument (`-` is typically used for this).

Reply:
Made the second argument required, and added support for the special value `-`, which is `stdin`/`stdout` for get-input/set-input respectively